includes/Admin/Admin_Page.php

```diff
@@ -30,13 +49,44 @@ public function add_hooks() {
 	 * @since n.e.x.t
 	 */
 	public function add_page() {
-		add_management_page(
+		$hook = add_management_page(
 			__( 'Plugin Check', 'plugin-check' ),
 			__( 'Plugin Check', 'plugin-check' ),
 			'activate_plugins',
 			'plugin-check',
 			array( $this, 'render_page' )
 		);
+
+		add_action( "load-{$hook}", array( $this, 'initialize_page' ) );
+	}
+
+	/**
+	 * Initializes page hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function initialize_page() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+	}
+
+	/**
+	 * Loads the check's script.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function enqueue_scripts() {
+		wp_enqueue_script( 'plugin-check-admin', WP_PLUGIN_CHECK_PLUGIN_DIR_URL . 'assets/js/plugin-check-admin.js', array(), '1.0.0', true );
```

Comment:
The version number on assets is used for cache invalidation, so following the WP core approach we should set this to the current plugin version just in case: `WP_PLUGIN_CHECK_VERSION`